static/src/stylesheets/module/identity/_identity.scss

```diff
@@ -275,6 +275,30 @@ $identity-header-height: 48px;
     }
 }
 
+/* Email Change
+   ========================================================================== */
+
+.form-field.form-field--email-change {
+    position: relative;
+    padding-left: 24px;
+
+    .inline-icon {
+        background-color: $neutral-3;
+        border-radius: 1000px;
+        fill: $neutral-1;
```

Comment:
we should be using the newer `$garnett-neutral-X`. Not sure what the matching neutrals are but there should be close ones